packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -253,6 +323,41 @@ class ContainerPlugin {
         );
       },
     );
+
+    // add include of federation runtime
+    compiler.hooks.thisCompilation.tap(
+      PLUGIN_NAME,
+      (compilation: Compilation, { normalModuleFactory }) => {
+        const federationRuntimeDependency =
```

Comment:
Attempt to reduce the number of instances of federation runtime dependency that are created. 
Multiple get created because new FederationRuntimePlugin() is called multiple times in several plugins, so there are more instances of the dependency created